src/util/mpsc/batch.rs

```diff
@@ -59,9 +59,9 @@ impl State {
         }
     }
 
-    // When the `Receiver` holds the `State` is running on an `Executor`, call this to yield from
-    // current `poll` context and put the current task handle to `recv_task` so that the `Sender`
-    // respectively can notify it after send some messages into the channel.
+    /// When the `Receiver` holds the `State` is running on an `Executor`, call this to yield from
```

Comment:
What does "When the `Receiver` holds the `State` is running on an `Executor`" mean? I don't understand the meaning of "holds".